common/v2/components/WalletList/WalletButton.tsx

```diff
@@ -78,26 +77,29 @@ const WalletLabel = styled.div`
 
 const WalletIcon = styled.img`
   max-height: 75px;
-  opacity: 0.8;
 
   @media screen and (max-width: ${SCREEN_SM}) {
     max-height: 60px;
   }
 `;
 
-export class WalletButton extends React.PureComponent<Props> {
-  public render() {
-    const { name, icon, margin } = this.props;
-
-    return (
-      <WalletButtonWrapper onClick={this.handleClick} margin={margin}>
-        {icon && <WalletIcon src={icon} alt={name + ' logo'} />}
-        <WalletLabel>{name}</WalletLabel>
-      </WalletButtonWrapper>
-    );
-  }
-
-  private handleClick = () => {
-    this.props.onClick(this.props.walletType);
-  };
-}
+export const WalletButton = (props: Props) => {
```

Comment:
Can we add a `story` for this please?

Reply:
Added story with two states (enabled/disabled).